### PR TITLE
Problem: Missing APY and commission sorting

### LINKED ISSUE #388: Problem: Missing APY and commission sorting

### PR DESCRIPTION
**Pull request by calvinaco**
Solution: (Fix #388) Add APY and commission sorting

### CODE REVIEW

File: infrastructure/cosmosapp/httpclient.go [hunk 1]
```diff
@@ -496,6 +496,78 @@ func (client *HTTPClient) Delegation(
...
+		rawRespBody, statusCode, err := client.rawRequest(queryUrl)
+		if err != nil {
+			return coin.Coin{}, err
+		}
+		defer rawRespBody.Close()
+
+		if decodeErr := jsoniter.NewDecoder(rawRespBody).Decode(&resp); decodeErr != nil {
+			return coin.Coin{}, decodeErr
+		}
+		if resp.MaybeCode != nil && *resp.MaybeCode == 2 {
```
Comment (cdc-Hitesh) [inline infrastructure/cosmosapp/httpclient.go:546]: Can we put a comment for `.MaybeCode value` `2`?

**Review by cdc-Hitesh — APPROVED** [review]
Overall looks good to me.

File: infrastructure/cosmosapp/httpclient.go (hunk 1 above)
Comment (calvinaco) [reply infrastructure/cosmosapp/httpclient.go:546]: Good point.

Comment (calvinaco) [reply infrastructure/cosmosapp/httpclient.go]: Done, update it to a constant. Code is better the comment 👍🏻

**Pull request merged**